Protobuf-cc.podspec

```diff
@@ -0,0 +1,107 @@
+                    'src/google/protobuf/repeated_field_unittest.cc',
+                    'src/google/protobuf/test_util.cc',
+                    'src/google/protobuf/test_util.h',
+                    'src/google/protobuf/test_util_lite.cc',
+                    'src/google/protobuf/test_util_lite.h',
+                    'src/google/protobuf/text_format_unittest.cc',
+                    'src/google/protobuf/unknown_field_set_unittest.cc',
+                    'src/google/protobuf/well_known_types_unittest.cc',
+                    'src/google/protobuf/wire_format_unittest.cc',
+                    'src/google/protobuf/stubs/bytestream_unittest.cc',
+                    'src/google/protobuf/stubs/common_unittest.cc',
+                    'src/google/protobuf/stubs/int128_unittest.cc',
+                    'src/google/protobuf/stubs/io_win32_unittest.cc',
+                    'src/google/protobuf/stubs/once_unittest.cc',
+                    'src/google/protobuf/stubs/statusor_test.cc',
+                    'src/google/protobuf/stubs/status_test.cc',
+                    'src/google/protobuf/stubs/stringpiece_unittest.cc',
+                    'src/google/protobuf/stubs/stringprintf_unittest.cc',
+                    'src/google/protobuf/stubs/structurally_valid_unittest.cc',
+                    'src/google/protobuf/stubs/strutil_unittest.cc',
+                    'src/google/protobuf/stubs/template_util_unittest.cc',
+                    'src/google/protobuf/stubs/time_test.cc',
+                    'src/google/protobuf/stubs/type_traits_unittest.cc',
+                    'src/google/protobuf/io/coded_stream_unittest.cc',
+                    'src/google/protobuf/io/printer_unittest.cc',
+                    'src/google/protobuf/io/tokenizer_unittest.cc',
+                    'src/google/protobuf/io/zero_copy_stream_unittest.cc',
+                    'src/google/protobuf/util/type_resolver_util_test.cc',
+                    'src/google/protobuf/util/time_util_test.cc',
+                    'src/google/protobuf/util/json_util_test.cc',
+                    'src/google/protobuf/util/field_mask_util_test.cc',
+                    'src/google/protobuf/util/field_comparator_test.cc',
+                    'src/google/protobuf/util/delimited_message_util_test.cc',
+                    'src/google/protobuf/util/message_differencer_unittest.cc',
+                    'src/google/protobuf/util/internal/json_stream_parser_test.cc',
+                    'src/google/protobuf/util/internal/default_value_objectwriter_test.cc',
+                    'src/google/protobuf/util/internal/json_objectwriter_test.cc',
+                    'src/google/protobuf/util/internal/protostream_objectsource_test.cc',
+                    'src/google/protobuf/util/internal/protostream_objectwriter_test.cc',
+                    'src/google/protobuf/util/internal/protostream_objectwriter_test.cc'
+
+  s.header_mappings_dir = 'src'
+  s.libraries = 'c++'
```

Comment:
Is this needed or does Xcode just get things right when things get added to the projects/workspaces?

Reply:
Looks like it's not required, will remove.